scripts/lib.star

```diff
@@ -861,15 +860,16 @@ def deploy_to_kubernetes_step(edition, is_downstream=False):
         ],
     }
 
+def edition_sfx(edition):      
```

Comment:
Not 100% sure if edition_sfx is a good name, because it's used specifically for the "enterprise2" suffix, while in other contexts we can also use the "enterprise" edition suffix. I find it confusing, because I have to read the function to realize it's only dealing with the enterprise2 edition.

Reply:
yea agree that using putting enterprise2 in edition but it's not treated the same was as oss & enterprise edition is super duper confusing, took me a long time to understand that enterprise2 is not really a new edition but only in some new steps in the enterprise pipeline?

Reply:
not sure what other name to use, pro_sfx ?)